frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -365,6 +377,31 @@ function renderItemName(item: ListItem) {
   return item.displayName;
 }
 
+function renderItemWrapper(content: ReactNode) {
+  return <HoverParent>{content}</HoverParent>;
+}
+
+function renderItemIcon(item: ListItem) {
+  if (item.type !== "metric") {
+    return null;
+  }
+
+  if (!item.description) {
+    return null;
+  }
+
+  return (
+    <PopoverWrapper>
```

Comment:
`Flex` from `metabase/ui` should work. We generally don't add new styled components now. Either mantine or CSS